bot/kodiak/evaluation.py

```diff
@@ -298,6 +292,31 @@ async def set_status(msg: str, markdown_content: Optional[str] = None) -> None:
     # we keep the configuration errors before the rest of the application logic
     # so configuration issues are surfaced as early as possible.
 
+    if repository.is_private and (
```

Comment:
I think we'll want an env variable to enable this functionality so that self hosting users don't have to deal with it and such that we don't immediately roll it out on next deploy